drivers/hmis/app/models/hmis/form/form_processor.rb

```diff
@@ -138,9 +138,9 @@ def store_assessment_questions!
     # Queue up job to store CE Assessment responses in the HUD CE AssessmentQuestions table
     # Rspec test isolation interferes with delayed job transaction
     if Rails.env.test?
-      ::Hmis::AssessmentQuestionsJob.perform_now(custom_assessment_id)
+      ::Hmis::AssessmentQuestionsJob.perform_now(custom_assessment_ids: custom_assessment_id)
     else
-      ::Hmis::AssessmentQuestionsJob.perform_later(custom_assessment_id)
+      ::Hmis::AssessmentQuestionsJob.perform_later(custom_assessment_ids: custom_assessment_id)
```

Comment:
This was throwing an exception https://green-river.sentry.io/issues/5354872029/?alert_rule_id=14733065&alert_type=issue&environment=staging&notification_uuid=3a979adf-b383-43a9-8cb3-ea208f3df0f0&project=4503977346662400&referrer=slack

Reply:
I was using perform_now in development so this wasn't caught. whoops!